libraries/botbuilder-ai/package.json

```diff
@@ -28,7 +28,7 @@
   },
   "dependencies": {
     "@azure/cognitiveservices-luis-runtime": "2.0.0",
-    "@azure/ms-rest-js": "1.9.1",
+    "@azure/ms-rest-js": "^2.6.1",
```

Comment:
IIRC this is might be a breaking change due to the underlying library using `fetch` instead of `axios`. Would not upgrade this without deciding it's okay to take a breaking change./

Reply:
@ceciliaavila Can we confirm if this is a breaking change and if there are alternatives?

Reply:
Hi @tracyboehrer,

We reviewed the code where this library is used and the [changes](https://github.com/Azure/ms-rest-js/blob/master/Changelog.md#200---2019-06-21) that its new versions introduced and consider that they are not breaking changes for us (we are not using the interfaces and methods that were updated).

We also tested the libraries with the _azure/ms-rest-js_ upgraded version:

- We published the packages to a testing feed and ran the [BotFramework-FunctionalTests](https://github.com/microsoft/BotFramework-FunctionalTests/tree/main/Tests) pipelines. All tests passed (cards, proactive, authentication (single-tenant, multi-tenant, MSI), among others).
- We tested a sample using _botframework-streaming_ and _webchat_ and it worked.
- We tested the _CoreBot_ sample connected to a LUIS app and it worked as expected.

However, as an alternative, we could leave this dependency as it was and update only _azure/identity_, and _jsonwebtoken_ versions.

Please, let us know if you have any questions.